src/oas.md

```diff
@@ -3077,52 +3077,100 @@ will map to `#/components/schemas/Dog` because the `dog` entry in the `mapping`
 #### XML Object
 
 A metadata object that allows for more fine-tuned XML model definitions.
-
-When using arrays, XML element names are _not_ inferred (for singular/plural forms) and the `name` field SHOULD be used to add that information.
-See examples for expected behavior.
+When using a Schema Object with XML, if no XML Object is present, the behavior is determined by the XML Object's default field values.
 
 ##### Fixed Fields
 
 | Field Name | Type | Description |
 | ---- | :----: | ---- |
-| <a name="xml-name"></a>name | `string` | Replaces the name of the element/attribute used for the described schema property. When defined within `items`, it will affect the name of the individual XML elements within the list. When defined alongside `type` being `"array"` (outside the `items`), it will affect the wrapping element if and only if `wrapped` is `true`. If `wrapped` is `false`, it will be ignored. |
+| <a name="xml-node-type"></a>nodeType | `string` | One of `element`, `attribute`, `text`, `cdata`, or `none`, as explained under [XML Node Types](#xml-node-types).  The default value is `none` if `$ref`, `$dynamicRef`, or `type: "array"` is present in the [Schema Object](#schema-object) containing the XML Object, and `element` otherwise. |
```

Comment:
I'm not 100% the bits about `$ref` and `$dynamicRef` are backwards-compatible, because I'm not sure what, if anything, the behavior was in those cases before, as "property schema" was not well-defined.  If we mean "literal inline schema directly under a name under `properties` without references", then there was no defined behavior and we can do what we want.

Otherwise, it might not be, and we should let schemas containing `$ref` or `$dynamicRef` have a default `nodeType` of `element`, and folks can explicitly set them to `none`. I don't think that change would impact anything else in this PR, and might be the more conservative compatibility choice.